### PR TITLE
Article: 12 Principles Of Agile Manifesto

### PR DESCRIPTION
**Pull request by SeanFCC**
Creates an article regarding the 12 Principles of the Agile Manifesto.

See #667

### CODE REVIEW

**Comment by Rafase282** [comment]
You forgot to save as `.md`

**Comment by SeanFCC** [comment]
Sorry about that. Fixed now.

File: 12-Principles-Of-Agile-Manifesto.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+6. The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.
+
+7. Working software is the primary measure of progress.
+
+8. Agile processes promote sustainable development. The sponsors, developers, and users should be able 
+to maintain a constant pace indefinitely.
+
+9. Continuous attention to technical excellence and good design enhances agility.
+
+10. Simplicity -- * the art of maximizing the amount of work not done * -- is essential.
```
Comment (Rafase282) [inline 12-Principles-Of-Agile-Manifesto.md]: Is this supposed to be bold or italics?

**Comment by abhisekp** [comment]
@SeanFCC which task is this or is it not listed in https://github.com/FreeCodeCamp/wiki/issues/667 ?

File: 12-Principles-Of-Agile-Manifesto.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+11. The best architectures, requirements, and designs emerge from self-organizing teams.
```
Comment (theflametrooper) [inline 12-Principles-Of-Agile-Manifesto.md]: requirements~~,~~ and designs

File: 12-Principles-Of-Agile-Manifesto.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# 12 Principles Behind The Agile Manifesto
+
+### We follow these principles:
+
+1. Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.
+
+2. Welcome changing requirements, even late in development. Agile processes harness change for the customer's competitive advantage.
```
Comment (theflametrooper) [inline 12-Principles-Of-Agile-Manifesto.md]: A different tense is used in Question 1 compared to the other questions.

File: 12-Principles-Of-Agile-Manifesto.md (hunk 2 above)
Comment (SeanFCC) [inline 12-Principles-Of-Agile-Manifesto.md]: I'm not sure about this one. Are you sure this change should be made? Also this was directly copied, it is open for anyone to use, directly from the agile manifesto website.

File: 12-Principles-Of-Agile-Manifesto.md (hunk 3 above)
Comment (SeanFCC) [inline 12-Principles-Of-Agile-Manifesto.md]: This was copied, it is open for all, directly from the source. Would it help, in your opinion, if I added the link to the original? My first thought was to keep them on our wiki but I can see linking to the source as a reference as well.

**Comment by theflametrooper** [comment]
@SeanFCC Because this was directly copied, an exception can be allowed. It would be good however, as you suggested, if you included a link the original source.

**Pull request merged**